src/aiortc/mediastreams.py

```diff
@@ -144,3 +149,33 @@ async def recv(self) -> Frame:
         frame.pts = pts
         frame.time_base = time_base
         return frame
+
+class KeypointsStreamTrack(MediaStreamTrack):
+    """
+    A dummy keypoints track which reads the constant keypoints.
+    """
+
+    kind = "keypoints"
+
+    _start: float
+    _timestamp: int
+
+    async def next_timestamp(self) -> Tuple[int, fractions.Fraction]:
+        if self.readyState != "live":
+            raise MediaStreamError
+
+        if hasattr(self, "_timestamp"):
+            self._timestamp += int(VIDEO_PTIME * VIDEO_CLOCK_RATE)
+            wait = self._start + (self._timestamp / VIDEO_CLOCK_RATE) - time.time()
+            await asyncio.sleep(wait)
+        else:
+            self._start = time.time()
+            self._timestamp = 0
+        return self._timestamp, VIDEO_TIME_BASE
```

Comment:
are these values necessary/where are they used?

Reply:
do you need this function or can you just defer to the base class and not reeimplement?